src/encoded/static/components/search.js

```diff
@@ -509,16 +521,39 @@ const ExperimentComponent = (props, reactContext) => {
                                 {synchronizations && synchronizations.length > 0 ?
                                     <div><span className="result-item__property-title">Synchronization timepoint: </span>{synchronizations.join(', ')}</div>
                                 : null}
-
+                                {elementsSelectionMethod && elementsSelectionMethod.length > 0 ?
+                                    <div><span className="result-item__property-title">Elements selection method: </span>{elementsSelectionMethod.join(', ')}</div>
+                                : null}
+                                {referenceLoci && referenceLoci.length > 0 ?
```

Comment:
To be consistent with other places in the repo, I changed this part to not include commas in the links.